website/source/api/acl/tokens.html.md

```diff
@@ -35,6 +35,12 @@ The table below shows this endpoint's support for
 
 ### Parameters
 
+- `AccessorID` `(string: "")` - Specifies a UUID to use as the token's Accessor ID.
+   If not specified a UUID will be generated for this field. Added in v1.5.0.
+
+- `SecretID` `(string: "")` - Specifies a UUID to use as the token's Secret ID.
+   If not specified a UUID will be generated for this field. Added in v1.5.0.
```

Comment:
Should we add suitable warnings here about entropy strength?

Reply:
Do you have any particular recommendations?

We could say:

> The SecretID is used to authorize operations against Consul and should be generated from an appropriate cryptographic source.

Maybe that is a random bit generator suitable for cryptographic use. Maybe its a the first 16 bytes of some cryptographic hash of a password you use. However you choose to generate it ensure that its secure.

Reply:
Yep that would be great. I like your wording.